molgenis-app/dev-env/filebeat.yml

```diff
@@ -0,0 +1,18 @@
+filebeat.inputs:
+  - type: log
+    json.message_key: message
+    json.keys_under_root: true
+    paths:
+      - /usr/share/filebeat/input/audit*.log
+processors:
+  - timestamp:
+      field: timestamp
+      layouts:
+        - '2006-01-02T15:04:05.999Z'
```

Comment:
am still a bit surprised that this worked :) have you checked the docs that this is how it's supposed to be done?

Reply:
https://www.elastic.co/guide/en/beats/filebeat/current/processor-timestamp.html

it's copied straight from the docs! 😎